### DISCUSSION TITLE
🔥 Hot Off the Press - Copilot Monthly Check-in

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#128090] [Copilot News and Announcements]
👋 We’re back with the latest updates from the world of AI 🌎 powered by GitHub Copilot! 

🔢 Let’s go!

## ✨ Community Champions: Amplifying the Resounding Shoutouts! 📢

- A big round of applause ✨ for @shgnplaatjies for sharing their expertise with the community, and enabling users to [sign up for Copilot](https://github.com/orgs/community/discussions/125623).
- Thank You @hafiz-muhammad for helping a user [manage their Copilot subscription](https://github.com/orgs/community/discussions/122713) and providing references to GitHub Docs. 📝
- Sparkles✨ for @RoccoKing98 for doubling down on [their issue logging into VSCode extension using the correct username](https://github.com/orgs/community/discussions/125441) and pulling off an amazing self-solve. 
- A brilliant self-solve 💡 by @lancewilhelm, first for asking a really descriptive question on broken [WSL autocomplete configuration with Copilot](https://github.com/orgs/community/discussions/123300) and then updating the community with the solution. 💗

> ✅ Mark as Answer: Don’t forget to click the **‘Mark as Answer**’ button when someone solves your question to easily help others who may have the same question ✅

### 👀  [Copilot Chat can now…](https://github.blog/changelog/2024-05-31-github-copilot-enterprise-copilot-knows-about-repositories-releases-commits-more-may-2024-updates)

Copilot Chat in GitHub.com has leveled up 🔋. It can now summarize and answer questions about your repositories, releases, commits and more.

- **Learn about repositories**: Copilot can answer questions about a repository or compare repositories based on its owner, description, stars & forks, and various timestamps. For details, see "Asking exploratory questions about a repository" in the GitHub docs.

> Try it yourself: Navigate to a repository on GitHub.com and ask Copilot, "Tell me about this repo."

- **Ask about releases**: Copilot can summarize or retrieve the most recent release in a repository. Ask Copilot to summarize the latest release from your favorite open source projects and explore issues or changes tied to the release notes. For more information, see "Asking exploratory questions about a repository" in the GitHub docs.

> Try it yourself: Navigate to an open source repository on GitHub.com and ask Copilot, "Tell me about the latest release."

- **Understand changes in your codebase**: Copilot can list changes (commits) related to a pull request and dive into specific commits for detailed understanding. For more details, see "Asking a question about a specific commit" in the GitHub docs.

> Try it yourself: Navigate to a commit on GitHub.com and ask Copilot, "Summarize the changes in this commit."

📌  [Learn more in our docs.](https://docs.github.com/en/copilot/github-copilot-chat/copilot-chat-in-github/about-github-copilot-chat-in-githubcom)

### 🎉  Bonus

> [!TIP] 
> Want a quick recap of what’s happening in the GitHub and AI community? Check out this video from [The Download](https://www.youtube.com/watch?v=SfXK8INAUSQ).  

### ⏭️  [Copilot Extensions now in Limited Public Beta](https://github.blog/changelog/2024-05-21-copilot-extensions-now-in-limited-public-beta)

Enhanced GitHub Copilot Chat can now perform actions across third-party tools and services. Create feature flags, check logs, access API docs, and deploy apps to the cloud—all via natural language. 🗨️ 

- Access to Copilot Extensions is currently limited but will expand gradually. Learn more in the [documentation](https://docs.github.com/en/copilot/github-copilot-chat/github-copilot-extensions/about-github-copilot-extensions) and [GitHub Marketplace](https://github.com/marketplace?type=apps&copilot_app=true). 
- To build a GitHub Copilot Extension, join the [Copilot Partner Program](https://github.com/features/preview/copilot-partner-program) to integrate your tools and services across supported clients. For VS Code-specific extensions, see the [documentation](https://code.visualstudio.com/api/extension-guides/chat).
- Questions or suggestions? Join the [community discussion](https://github.com/orgs/community/discussions/125067).

### 🆕  [Updates to Copilot Metrics](https://github.blog/changelog/2024-05-09-github-copilot-metrics-updates) 

We’ve updated how we calculate Last Activity for better clarity and paused access to the Team endpoint in the Metrics API.
Previously, Last Activity indicated when a user generated a Copilot authentication token, not actual engagement. Now, it reflects deliberate user actions such as code completions, chatting with Copilot, updating knowledge bases, creating pull request summaries, and interacting with Copilot on GitHub.com. This provides more accurate usage data for admins.[ Learn more here](https://github.blog/changelog/2024-05-09-github-copilot-metrics-updates).

## 💎  World’s fair of software - [GitHub Universe 2024](https://githubuniverse.com/) is Back! 🛰️ 

This is really exciting! GitHub Universe is back 🔙  and we are thrilled to announce the in-person and virtual event this October 29-30. So, don’t forget to reserve your spots. 🥳

🎥  [Gear up for GitHub Universe 2024: a celebration of code and community](https://www.youtube.com/watch?v=hKgTC6d11w4)

📅 Save the date → October 29-30
📍 In-person in San Francisco or online.👉  3,500+ Attendees → 150+ Speakers→ 100+ Sessions→ 35+Featured partners → on AI, Security, and DevEx

Super early bird 💸  tickets are available now until July 8! [Get your tickets now!](https://reg.githubuniverse.com/flow/github/universe24/Registration/login)

## Resources to bookmark 🔖

- [4 ways GitHub engineers use GitHub Copilot](https://github.blog/2024-04-09-4-ways-github-engineers-use-github-copilot/)
- [Using GitHub Copilot in your IDE: Tips, tricks, and best practices](https://github.blog/2024-03-25-how-to-use-github-copilot-in-your-ide-tips-tricks-and-best-practices/)
- [GitHub Copilot Learning Pathway: Accelerate your business with AI](https://github.blog/2024-03-04-github-copilot-learning-pathway-accelerate-your-business-with-ai/)
- [Customizing and fine-tuning LLMs: What you need to know](https://github.blog/2024-02-28-customizing-and-fine-tuning-llms-what-you-need-to-know/)
- [Essentials of GitHub Copilot](https://resources.github.com/learn/pathways/copilot/essentials/essentials-of-github-copilot/)
- [10 unexpected ways to use GitHub Copilot](https://github.blog/2024-01-22-10-unexpected-ways-to-use-github-copilot/)

That's a wrap 📕  for this month's check-in. What’s your favorite Copilot feature or trick that you love using? Share it in the comments below. Happy Copiloting – Happy Coding! 🚀

**Comment by hafiz-muhammad** [comment]
Thank you, glad to have helped. 😊
